app/src/main/java/ch/epfl/sweng/zuluzulu/Utility/Utils.java

```diff
@@ -95,7 +95,7 @@ static long getMillisecondsSince(Date date) {
      * @param container The container view
      */
     static void showConnectSnackbar(View container) {
-        Snackbar.make(container, "Login to access more features ⭐", 5000).show();
+        Snackbar.make(container, "Connecte-toi pour avoir accès à toutes les options", 5000).show();
```

Comment:
You should put this in Strings.xml

Reply:
Right

Reply:
https://developer.android.com/guide/topics/resources/string-resource


https://developer.android.com/guide/topics/resources/string-resource#formatting-strings

Reply:
A Snackbar is a kind on xml "textview" that you fill with text.

Reply:
Dans ce cas on peut passer au crible tout le reste du projet parce que y'a des strings comme ça un peu partout